feed.xml

```diff
@@ -10,14 +10,14 @@ layout: none
         <description>{{ site.description | xml_escape }}</description>
         <link>{{ site.url }}</link>
         <atom:link href="{{ "/feed.xml" | prepend: site.url }}" rel="self" type="application/rss+xml"/>
-        <pubDate>{{ site.time | date_to_rfc822 }}</pubDate>
+        <lastmod>{{ site.time | date: '%Y-%m-%d' }}</lastmod>
```

Comment:
Do we have access to the publication date? Because if yes, then let’s use that.

Reply:
I dont know how to get it, so for now, i added the last modified date which works.
let me know if you know how to get the pubDate